vital/types/protobuf/detected_object.proto

```diff
@@ -41,7 +41,7 @@ message detected_object {
 
   required double confidence = 1;
   required bounding_box bbox = 2;
-  optional class_map classifcations = 3;
+  optional detected_object_type classifcations = 3;
```

Comment:
How hard would it be to correct this?

Reply:
Looks like it's only in the protobuf stuff, so "pretty easy"?

Reply:
Done.